connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -150,9 +150,11 @@
     public static final String ERRORS_LOG_INCLUDE_MESSAGES_CONFIG = "errors.log.include.messages";
     public static final String ERRORS_LOG_INCLUDE_MESSAGES_DISPLAY = "Log Error Details";
     public static final boolean ERRORS_LOG_INCLUDE_MESSAGES_DEFAULT = false;
-    public static final String ERRORS_LOG_INCLUDE_MESSAGES_DOC = "Whether to the include in the log the Connect record that resulted in " +
-            "a failure. This is 'false' by default, which will prevent record keys, values, and headers from being written to log files, " +
-            "although some information such as topic and partition number will still be logged.";
+    public static final String ERRORS_LOG_INCLUDE_MESSAGES_DOC = "Whether to the include in the log the Connect record that resulted in a failure." +
+            "For sink records, the topic, partition, offset, and timestamp will be logged. " +
+            "For source records, the key and value (and their schemas), all headers, and the timestamp, Kafka topic, Kafka partition, source partition, " +
```

Comment:
What do you think about clarifying even more precisely that it's the Kafka topic _name_ and Kafka partition _number_? I know, it's nonsensical to log the topic or partition _contents_, but someone that is still learning how Connect works might have fewer questions. Plus, there is other topic metadata (like [topic IDs](https://cwiki.apache.org/confluence/display/KAFKA/KIP-516%3A+Topic+Identifiers)) that we're not logging.

Reply:
High-level, I think this description is already verbose enough and if anything, it should be shortened.

RE "topic name", I think that restricting the definition to just topic names and not identifiers is unnecessary and puts us in an awkward decision if it's ever desirable to include that information in the logged error messages since it's not obvious in that part of the code base that this docstring would need to be updated.

RE "partition number", I think that "partition" is all but completely unambiguous and that suffixing it with "number" is redundant.

Reply:
The change looks good! Much more clear.
One small thing... should "Whether to the include"  be "Whether to include"?